src/transformers/training_args.py

```diff
@@ -2397,6 +2404,9 @@ def set_push_to_hub(
                 with `huggingface-cli login`.
             private_repo (`bool`, *optional*, defaults to `False`):
                 If True, the Hub repo will be set to private.
+            push_all (`bool`, *optional*, defaults to `False`):
```

Comment:
I would rename it `always_push` (and `hub_always_push`) instead of `push_all` to emphasize that it affects the number of commits and not the content of the commits. But feel free to ignore.